packages/govern/internal/clients/graphql/GraphQLClient.ts

```diff
@@ -0,0 +1,103 @@
+import fetch from 'isomorphic-unfetch'
+import { Client } from '@urql/core'
+import { DocumentNode } from 'graphql'
+import ClientInterface from '../lib/ClientInterface'
+
+/**
+ * TODO: Use QueryResult type from govern-server/core
+ *
+ * @class GraphQLClient
+ */
+export default class GraphQLClient implements ClientInterface {
+  /**
+   * The urql/core Client object
+   *
+   * @var {Client} client
+   *
+   * @private
+   */
+  private client: Client
+
+  /**
+   * @param {string} governUrl
+   *
+   * @constructor
+   */
+  constructor(private governUrl: string) {
+    this.connect()
+  }
+
+  /**
+   * Initiates the Client from urql/core
+   *
+   * @method connect
+   *
+   * @returns {void}
+   */
+  private connect(): void {
+    this.client = new Client({
+      maskTypename: true,
+      url: this.governUrl,
+      fetch
+    })
+  }
+
+  /**
+   * Executes the query against the connected Govern server
+   *
+   * @method request
+   *
+   * @param {DocumentNode} query
+   * @param {Object} args
+   *
+   * @returns {Promise<any>}
+   *
+   * @public
+   */
+  public async request(query: DocumentNode, args: any = {}): Promise<any> {
```

Comment:
I think we could import and use [`OperationResult`](https://github.com/aragon/govern/blob/graphql-api/packages/govern-server/src/core/types.ts#L1) to type the returned value here.